src/dataprotection/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.8.0'
+VERSION = '0.8.1'
 try:
     from azext_dataprotection.manual.version import VERSION
 except ImportError:
```

Comment:
You can remove the `manual.version` file and maintains `VERSION` only in the setup.py file.

Reply:
Updated. `VERSION` now maintained only in setup.py